tests/execute/result/special.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Seems that `fmf` test metadata are missing, thus the test script is not executed.